frontend/public/components/build-config.tsx

```diff
@@ -61,6 +61,7 @@ const menuActions = [
 
 export const BuildConfigsDetails: React.SFC<BuildConfigsDetailsProps> = ({obj: buildConfig}) => <React.Fragment>
   <div className="co-m-pane__body">
+    <h1 className="co-section-title">Build Config Overview</h1>
```

Comment:
I think this should be `co-m-pane__heading` (not that we're exactly consistent).  We'll get that cleaned up with https://jira.coreos.com/browse/CONSOLE-570